service.yaml

```diff
@@ -1 +1,6 @@
 name: argocd-bot
+tests:
```

Comment:
Probably don't need this at all if the tests are in the build, since that will run prior to CI, and fail the build if it fails.

Another option would be to pull the tests out into here, but I don't know how much work that is.